src/custom-elements/src/components/expansion-panel/expansion-panel.tsx

```diff
@@ -0,0 +1,89 @@
+import { Component, Event, EventEmitter, Prop } from '@stencil/core';
+
+@Component({
+    tag: 'chi-expansion-panel',
+    styleUrl: 'expansion-panel.scss',
+    scoped: true
+})
+export class ExpansionPanel {
+    /**
+     * to set expansion panel to 'done' state
+     */
+    @Prop({ reflectToAttr: true }) done?: boolean;
```

Comment:
This attributes are not optional. They are boolean so they are true if the attribute exists or false if it doesn't. They are false by default. Apply the same rule to the next boolean attributes. 

In another point of view, done, active and disabled properties might be configured by only one property called state that accepts values: `pending`, `active`, `done` and `disabled`. This second approach looks better to me.

Reply:
My fault, @dani-cl-madrid, I thought not having to include explicitly a value made them optional but you are right ;)

I agree with the suggested change for _state_ property. Thanks!